app/src/pages/inside/logsPage/logToolbar/logToolbar.scss

```diff
@@ -18,3 +18,31 @@
   display: inline-block;
   margin-right: -1px;
 }
+
+@media (min-width: $SCREEN_SM_MIN) {
+  .action-button:first-child {
+    padding-left: 53px;
+  }
+}
+@media (max-width: $SCREEN_XS_MAX_DASHBOARD) {
```

Comment:
Is this breakpoint used for dashboard here?

Reply:
No, but this constant value fits the needs of this fix the most. Should I create new constants for those that I can't find a match for?